martin/src/srv/cors.rs

```diff
@@ -1,6 +1,15 @@
 use actix_http::Method;
+use log::info;
 use serde::{Deserialize, Serialize};
 
+use crate::{MartinError, MartinResult};
+
+#[derive(thiserror::Error, Debug)]
+pub enum CorsError {
+    #[error("Invalid CORS configuration")]
+    PropertyValidationError(String),
```

Comment:
If we only have one error, then keeping it overly generic as an `PropertyValidationError(String)` is not quite nessesary.

```suggestion
    #[error("At least one 'origin' must be specified in the 'cors' configuration")]
    NoOriginsConfigured,
```